### PR TITLE
Changelog v0.15.0

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,9 +25,10 @@
   * Add repo permission endpoint (#436)
   * Add web-config endpoint (#433)
   * Replace www-path with www-proxy option for development (#248)
-* SECURITY
-  * Add linter bidichk to prevent malicious utf8 chars (#516)
 * BUGFIXES
+  * make gRPC error "to many keepalive pings" only show up in trace logs (#787)
```
Comment (jolheiser) [inline CHANGELOG.md:29]:
```suggestion
  * Make gRPC error "too many keepalive pings" only show up in trace logs (#787)
```

**Review by jolheiser — APPROVED** [review]
LGTM, small nit

File: CHANGELOG.md [hunk 2]
```diff
@@ -64,6 +49,18 @@
   * Append trailing slash to default GH API URL (#411)
   * Fix filter pipeline config files (#279)
 * ENHANCEMENTS
+  * View better error if repo was deleted/renamed (#780)
+  * Add support for default clone image environment variable (#769)
```
Comment (anbraten) [inline CHANGELOG.md:53]:
```suggestion
  * Add support to set default clone image via environment variable (#769)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -64,6 +49,18 @@
   * Append trailing slash to default GH API URL (#411)
   * Fix filter pipeline config files (#279)
 * ENHANCEMENTS
+  * View better error if repo was deleted/renamed (#780)
```
Comment (anbraten) [inline CHANGELOG.md:52]:
```suggestion
  * Return better error if repo was deleted/renamed (#780)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -64,6 +49,18 @@
...
+  * Add support for default clone image environment variable (#769)
+  * Add flag to always authenticate when cloning public repositories (#760)
```
Comment (anbraten) [inline CHANGELOG.md:54]:
```suggestion
  * Add flag to always authenticate when cloning public repositories from locked down / private only forges (#760)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -64,6 +49,18 @@
...
+  * UI: show date time on hover over time items (#756)
+  * Add repo-link to badge markdown in UI (#753)
+  * Allow specifying dind container in values (#750)
+  * Add page to view all projects of a user / group (#741)
+  * Let non required migration tasks fail and continue (#729)
+  * Improve pipeline compiler (#699)
+  * Support ChangedFiles for Github & Gitlab PRs and Gitea pushes (#697)
```
Comment (anbraten) [inline CHANGELOG.md:61]:
```suggestion
  * Support ChangedFiles for Github & Gitlab PRs and pushes and Gitea pushes (#697)
```

**Review by anbraten — APPROVED** [review]

**Pull request merged**